homeassistant_api/rawbaseclient.py

```diff
@@ -73,20 +79,32 @@ def prepare_get_entity_histories_params(
         end_timestamp: Optional[datetime] = None,
         significant_changes_only: bool = False,
     ) -> Tuple[Dict[str, Optional[str]], str]:
-        """Pre-logic for `Client.get_entity_histories` and `Client.async_get_entity_histories`."""
+        """
+        Pre-logic for :py:meth:`Client.get_entity_histories` and :py:meth:`Client.async_get_entity_histories`.
+
+        Ensure timestamps
+
```

Comment:
This blank line is here so the bullet list renders correctly (it formats as a paragraph with asterisks in it without the empty line here)